### DISCUSSION TITLE
Playback skip forward/backwards values.

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#12309] [Ideas]
When playing back recordings the option to skip forward or backwards are both set to 10 seconds. Ideally these values shoud be different for example 15s forward 7s backward this allows for fine tuning a section of the playback video otherwise you skip 10s and go back 10s takes you back to the same point.
Version 14 beta 3.

**Comment by Duskspeed** [comment]
I do miss the "go to dectection" from v13, the best solution is atleast on desktop (or desktop mode on android) for me is to use the player controls and scrub the recording manualy, it gives much more granular control than the +/-10 seconds buttons or the side timeline; which, if I'm being honest I haven't used outside of just scrolling through large chunk of time and even then I'm more likely to just hit the back button and find another detection event since v14 came out for a number of reasons, seeking usability is poor when compared to other softwares (Synology's DScam on android or their desktop client in my usecase) timeline scrubbing. things like it not having "pinch zoom" or auto zoom and also I personally just don't like it being vertical, it takes up way too much room, on android  it takes up 1/2 of the screen just for the timeline.


RIghtclick on video -> Show all controls

![image](https://github.com/blakeblackshear/frigate/assets/103481526/6ebdff70-c1dd-4947-9f71-8a8c54de5651)

Without All controls
![image](https://github.com/blakeblackshear/frigate/assets/103481526/095e1a4e-62c5-4f98-8650-ce2656999554)

With all controls
![image](https://github.com/blakeblackshear/frigate/assets/103481526/af8a402b-cf62-406b-b425-2dc8a9665adf)

**Comment by 18923** [comment]
Is it possible to set playback skip forward/backward values in the config files? I would like to make the 2 values different.

**Reply by hawkeye217** [reply]
No, this is not possible. We are working on improved seeking and scrubbing for a future version.
